kube-monitoring/charts/templates/_helper.tpl

```diff
@@ -129,7 +129,6 @@ release: {{ $root.Release.Name | quote }}
 {{- define "kubeMonitoring.dashboardSelectorLabels" }}
 {{- $path := index . 0 -}}
 {{- $root := index . 1 -}}
-plugin: {{ $root.Release.Name }}
```

Comment:
why do we need to remove this?

Reply:
Ok, now that I look at the labels, it looks like we have the same helper twice:

* [kube-prometheus-stack.labels](https://github.com/cloudoperators/greenhouse-extensions/blob/main/kube-monitoring/charts/templates/_helper.tpl#L30-L39) 
* [kubeMonitoring.labels](https://github.com/cloudoperators/greenhouse-extensions/blob/main/kube-monitoring/charts/templates/_helper.tpl#L116-L127)

Do you comply? I think we should split this in 2 PRs. keep this one about the blackbox config change and open a new one for the duplicate labels, where we can remove the helper.

Reply:
agreed